_includes/certification.html

```diff
@@ -0,0 +1,24 @@
+<aside class="certification-section content--blue">
+  <div class="certification-section__content">
+    <div>
+      <img src="/img/smv_digital.svg" alt="SMV:Digital" class="certification-section__content-logo" loading="lazy" />
+      <p>
+        As registered advisors at
+        <a href="https://smvdigital.dk/content/ydelser/tilbud/4e84e652-3d63-4b24-b5f5-a278d2ad27e6/" target="_blank" rel="noopener">SMV:Digital</a>
```

Comment:
Let's not have target="_blank" here.
https://www.w3.org/WAI/WCAG21/Techniques/html/H83.html

Reply:
The same for the rest of the logo links.